includes/descriptionList.py

```diff
@@ -45,6 +45,10 @@ def loadCSV(typ, idField):
 				# only import rows with an integer as id, allow subrics though
 				if re.match("^[0-9]+[A-D]?$", row[idField], re.IGNORECASE):
 					try:
+						if len(row[idField]) > 7:
```

Comment:
Wieso prüfst du, ob die Länge größer sieben ist? Stört dieser Code sonst in den anderen CSV-Dateien?